api-reference/general-tools/assessment/initialization/initialize-assessment.yml

```
@@ -38,132 +41,50 @@ paths:
                   }
                 }
 
+
       parameters:
-        - name: locale
-          in: header
-          description: >-
-            The locale of the user. It is used to determine the language of the assessment.
-          schema:
-            type: string
-          example: en
         - name: auth
           in: header
+          required: True
           description: >-
             The auth token of the user. It is used to authenticate the user.
           schema:
             type: string
           example: >-
             eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJiX2lkIjoiMTIzNDU2IiwiY2xpZW50X2lkIjoiNzg5MCIsImV4dHJhX2ZpZWxkIjoiZXh0cmFfZmllbGRfZGF0YSJ9.q9KzBI6f4l3OyM_EkB5Quq0l9EEMFh5JS-fx3F_PHUM
-        - name: Content-Type
+        - name: client_id
           in: header
+          required: True
           description: >-
-            The content type of the request. It is used to determine the type of data being sent.
+            The client id of the user. It is used to identify the user.
           schema:
             type: string
           example: application/json
+        - name: locale
+          in: header
+          description: >-
+            The locale of the user. It is used to determine the language of the assessment. Supported locales are `en`, `hi`, `kn` for english, hindi and kannada respectively. Default value is `en`.
+          schema:
+            type: string
+          example: en
       responses:
         '200':
           description: OK
-          headers:
-            Date:
-              schema:
-                type: string
-                example: Mon, 12 Aug 2024 07:41:05 GMT
-            Server:
-              schema:
-                type: number
-                example: WSGIServer/0.2 CPython/3.9.17
-            Content-Type:
-              schema:
-                type: string
-                example: application/json
-            Vary:
-              schema:
-                type: string
-                example: Accept, Cookie, origin
-            Allow:
-              schema:
-                type: string
-                example: POST, OPTIONS
-            X-Frame-Options:
-              schema:
-                type: string
-                example: DENY
-            Content-Length:
-              schema:
-                type: integer
-                example: '51'
-            X-Content-Type-Options:
-              schema:
-                type: string
-                example: nosniff
-            Referrer-Policy:
-              schema:
-                type: string
-                example: same-origin
-            Cross-Origin-Opener-Policy:
-              schema:
-                type: string
-                example: same-origin
           content:
             application/json:
               schema:
                 type: object
               example:
                 data:
                   init_hash: sn_1212121212
-        '400':
+        '5XX':
```

```suggestion
        '4XX':
```

5XX says internal server error's please have a look make changes accordingly

---

We have not finalized the error response yet. Will update it later once the error dictinary format and error codes gets finalized.
This one is written just to give an idea to frontend team (Gautham).